ci/mobile.groovy

```diff
@@ -51,6 +51,12 @@ def prep(type = 'nightly') {
   sh 'cp patches/js_realm.hpp node_modules/realm/src/js_realm.hpp'
   /* install ruby dependencies */
   sh 'bundle install --quiet'
+  /* download status-go for mvn */
+  def pomDir = 'modules/react-native-status/ios/RCTStatus'
+  def repo = "${pwd}/${pomDir}/repo/status-im/status-go-ios/1.15.0"
```

Comment:
`1.15.0`: you mean `0.15.0` right?

also, ideally there would be a single line needing to be updated with this by the `scripts/update-status-go`.

Reply:
That's true, consider this a work in progress. I still have a long way to go to make this work with `pom.xml`. It's so painful.